egs/multi_zh-hans/ASR/zipformer/export.py

```diff
@@ -0,0 +1,541 @@
+#!/usr/bin/env python3
```

Comment:
Can we replace it with a symlink to the file in wenetspeech?

Reply:
I slightly modified default arguments of this scripts from the original version, maybe we can leave this updated one here?